tc-package.sh

```diff
@@ -39,3 +39,5 @@ pixz -9 ${TASKCLUSTER_ARTIFACTS}/native_client.tar ${TASKCLUSTER_ARTIFACTS}/nati
 rm ${TASKCLUSTER_ARTIFACTS}/native_client.tar
 
 cp ${DS_ROOT_TASK}/DeepSpeech/ds/data/alphabet.txt ${TASKCLUSTER_ARTIFACTS}/
+cp ${DS_ROOT_TASK}/DeepSpeech/ds/data/lm/lm.binary ${TASKCLUSTER_ARTIFACTS}/
+cp ${DS_ROOT_TASK}/DeepSpeech/ds/data/lm/trie ${TASKCLUSTER_ARTIFACTS}/
```

Comment:
Is there value in packaging those ? I'm also questioning that for the alphabet (I know I asked for that), but now that we requires Git-LFS always, maybe we could drop that requirement ? Hosting that is non free, even though it should not be a big deal, but the language model files are quite big, so maybe we should avoid that. And the question holds with both PR and merge: I think it could make sense for merges, though, especially when we instruct people that by default there is no reason to rebuild them. It would, also, maybe make sense to just package that *with* the model, and avoid multiplicating the sources for people who need that, especially since there might be a tight relationship between model and language model.

Reply:
There's no tight relationship between model and language model. I don't have an opinion on packaging this or not, we'll have to include them in the release anyway. I guess I can just get those files from ~/DeepSpeech/ds instead of packaging them for the tests. I've made that change.